src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -31,7 +32,9 @@ object DplaMapData {
 case class DplaMapData(
                          sourceResource: DplaSourceResource,
                          edmWebResource: EdmWebResource,
-                         oreAggregation: OreAggregation
+                         oreAggregation: OreAggregation,
+                         // Add id property at the root level (following ES schema)
+                         id: ExactlyOne[String]
```

Comment:
I think (given my assumption that the IDs are URIs) that this may already be handled in one of the URI fields already there.